libs/common/utilities/src/css/layout/padding/stories/vl-padding.stories.ts

```diff
@@ -0,0 +1,90 @@
+import { css } from 'lit';
+import { html } from 'lit-html';
+import vlPaddingStoriesDoc from './vl-padding.stories-doc.mdx';
+
+export default {
+    id: 'styles-next-layout-afnemers-padding',
+    title: 'Styles-next/Layout (afnemers)/padding',
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            page: vlPaddingStoriesDoc,
+        },
+    },
+};
+
+const paddingCss = css`
+    .block {
+        width: 300px;
+        height: 100px;
+        background-color: lightblue;
+        padding: var(--vl-spacing--medium) 0; /* wordt via vl-region gezet */
+    }
+
+    .block__inner {
+        width: 100%;
+        height: 100%;
+        background-color: mediumspringgreen;
+        border: lightseagreen 2px solid;
+    }
+`;
+
+export const PaddingDefault = ({}) => html`
+    <style>
+        ${paddingCss}
+    </style>
+    <div class="block">
+        <div class="block__inner">&nbsp;default&nbsp;</div>
+    </div>
+`;
+PaddingDefault.storyName = 'vl-padding - default';
+
+export const PaddingSmall = ({}) => html`
+    <style>
+        ${paddingCss}
+    </style>
+    <div class="block vl-padding-next--small">
+        <div class="block__inner">&nbsp;vl-padding-next--small&nbsp;</div>
```

Comment:
Conform de `margin`-stories, zou de tekst `vl-padding-next--small` in buitenste element moeten staan, gezien dat het element is waarop de `vl-padding-next--small`-klasse is toegepast.

Ik zou ook ervoor kiezen, om de kleuren te inverteren: het element waarop de styling is toegepast in het lichtgroen, en het binnenste element dan in het donkergroen.

bv.: 
<img width="394" alt="image" src="https://github.com/user-attachments/assets/5bf5efa6-e817-4d49-b18e-cd87bb94d8d6">

(Heb code wijzigingen in stash staan).

Reply:
Wordt mss herzien maar niet op vorige manier.